src/handlers/RF24StreamLogHandler.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file RF24StreamLogHandler.h
+ *
+ * Created on: 2 Oct 2020
+ *     Author: Witold Markowski (wmarkow)
+ *
+ * Copyright (C)
+ *    2020        Witold Markowski (wmarkow)
+ *
+ * This General Public License does not permit incorporating your program into
+ * proprietary programs.  If your program is a subroutine library, you may
+ * consider it more useful to permit linking proprietary applications with the
+ * library.  If this is what you want to do, use the GNU Lesser General
+ * Public License instead of this License.
+ */
+
+#ifndef SRC_HANDLERS_RF24STREAMLOGHANDLER_H_
+#define SRC_HANDLERS_RF24STREAMLOGHANDLER_H_
+
+#include <RF24LogHandler.h>
+#include <Print.h>
+
+/**
+ * A log handler implementation which outputs log messages to a stream.
+ */
+class RF24StreamLogHandler : public RF24LogHandler
+{
+protected:
+   Print *stream;

```

Comment:
I needed to change this to ```Print``` (to match the included Print.h). Otherwise I started to have compilation issues.
Maybe the variable should be also changed from ```stream``` to ```print```?

Reply:
I like the way it is; `stream` is a bit more descriptive in this context.

Reply:
ok, let's leave ```stream```.